src/FakeItEasy/CompletedFakeObjectCallExtensions.cs

```diff
@@ -25,14 +25,7 @@ public static class CompletedFakeObjectCallExtensions
         [SuppressMessage("Microsoft.Design", "CA1011:ConsiderPassingBaseTypesAsParameters", Justification = "The compiler would not be able to figure out the type.")]
         public static IEnumerable<ICompletedFakeObjectCall> Matching<TFake>(this IEnumerable<ICompletedFakeObjectCall> calls, Expression<Action<TFake>> callSpecification)
         {
-            var factory = ServiceLocator.Current.Resolve<IExpressionCallMatcherFactory>();
-            var callExpressionParser = ServiceLocator.Current.Resolve<ICallExpressionParser>();
-            var matcher = factory.CreateCallMatcher(callExpressionParser.Parse(callSpecification));
-
-            return
-                from call in calls
-                where matcher.Matches(call)
-                select call;
+            return Matching(calls, callSpecification);

```

Comment:
> That was quick. I did think it went to my private repository first :)

It does! The code is still only in your repo. But since you have a PR open on that branch, Github immediately updates the PR.



> Not sure if casting is preferable to another methodname?

Both ways are fine by me :+1: